include/ftl/array_iterator.hpp

```diff
@@ -84,6 +86,10 @@ class array_iterator final : public iterator_interface<array_iterator<Item, N>,
     return *this;
   }
 
+  auto predecrement_impl() const -> void {
```

Comment:
This is what decrement operator declaration looks like inside a class, as defined by the standard:
T& T::operator--();

Reply:
fixed